src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1194,6 +1195,7 @@ void lv_vg_lite_flush(struct _lv_draw_vg_lite_unit_t * u)
 
     LV_VG_LITE_CHECK_ERROR(vg_lite_flush());
     u->flush_count = 0;
+    u->letter_count = 0;
```

Comment:
Was this left over by mistake?

Reply:
Nice catch! It should be located at `lv_vg_lite_finish`.